### DISCUSSION TITLE
V1 컨트롤러 MoveToPosition 함수 출발시 딜레이가 걸려요!

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#653] [Issue Report (Legacy)]
V1 컨트롤러에서 MoveToPosition 함수를 사용하면 움직임 시작시에 큰 딜레이가 걸려요 :)

<img width="566" alt="스크린샷 2022-04-22 오후 4 19 54" src="https://user-images.githubusercontent.com/90679926/164626187-94eba0d7-93f1-41b6-b05b-ac8b7e00fe61.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 월드 1.2.0 기준으로 테스트했을때는 딜레이가 발생하지 않습니다.
관련 코드나 영상 첨부 가능할까요?

https://user-images.githubusercontent.com/100118650/165670278-985d172b-fbc9-4942-a35f-38fc572d7a4f.mov

**Reply by Deytooh** [reply]
Joystick으로 해도 딜레이 없어요 ? 저도 있어요 딜레이 ㅠㅠ

**Reply by x4cf** [reply]
저는 조작계통을 전부 다시 제작하는것으로 해결해서 영상이 없지만, V1 조이스틱 사용시 움직이려고하면 아주 잠깐 멈춰있다 움직이기 시작하는 딜레이가 걸리긴 했었어요.
테스트할때는 대략 이런 코드였던것같아요 :)

ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.MoveToPosition(new Vector3(7, 0, 7));

**Reply by Zepeto-TechEvangelist** [reply]
조이스틱을 사용한 이동 상태 변환시 transition time이 적용되고 있고 정상적인 상황입니다. 
다만 사용자에 따라서 체감하는 딜레이 정도가 다른것 같아, 
추후에 이 트렌지션 타임을 커스마이즈 할 수 있게 제공될 예정입니다. 
감사합니다.

https://user-images.githubusercontent.com/100118650/165687137-4c7265aa-50aa-46ad-9664-a263ba6e76d1.mov

**Reply by Cha-SeongHun** [reply]
![Honeycam 2022-04-29 11-51-14](https://user-images.githubusercontent.com/100827088/165878411-98a43bf3-8762-4074-a2d5-eb79ac3ba705.gif)
올려주신 영상과는 조금 다른 느낌으로 부자연스러운 딜레이가 있는듯합니다.
참고해주시면 감사하겠습니다

**Reply by Zepeto-TechEvangelist** [reply]
딜레이 이슈는 내부 개선 사항으로 진행 중에 있습니다.
참고 부탁 드립니다.
감사합니다.
